libs/components/forms/testing/src/checkbox/fixtures/checkbox-harness-test.component.html

```diff
@@ -28,7 +28,7 @@
     <sky-checkbox
       data-sky-id="my-phone-checkbox"
       formControlName="phone"
-      helpKey="helpKey.html"
+      [helpKey]="helpKey"
```

Comment:
Since we're splitting the inline help button into three separate buttons (help key, popover, and user-defined) we can't always set the `helpKey` for the unit tests. Made some adjustments to the fixture so we can decide when the `helpKey` is set for the expectations.

(You'll see similar changes to other specs, below.)